src/test/mod.rs

```diff
@@ -312,6 +335,10 @@ impl TestDatabase {
             .get()
             .expect("failed to get a connection out of the pool")
     }
+
+    pub(crate) fn repository_stats_updater(&self) -> &RepositoryStatsUpdater {
+        &self.repository_stats_updater
+    }
```

Comment:
This function is useless, I would remove it and use the field directly.

Reply:
Another forgotten relic I guess. Good catch! Removing it.

Reply:
Actually, it's better this way. For example it allows to keep 

```rust
fn repository_stats_updater(&self) -> Result<Arc<RepositoryStatsUpdater>, Error> {
    Ok(self.repository_stats_updater())
}
```

as is in `TestEnvironment` instead of instancing the `Arc` manually. Also, in case we have a mutable access over the struct, it prevents to edit the field, adding another layer of "security".

Reply:
Hmm, ok. I don't feel super strongly about it.